code/API_definitions/EdgeCloud_LcM.yaml

```diff
@@ -0,0 +1,703 @@
+    # Introduction
+    APIs defined in this version of the specification can be categorized into the following areas:
+    * __Application Management__ - Deploy and remove applications within artifacts
+    * __Application Artifacts Management__ - Management of application descriptors, binaries, charts and packages 
+    * __MEC Platform information__ - Retrieves all the Edge nodes available according to some defined parameters
+    
+    # Relevant terms and definitions
+    This section provides definitions of terminologies commonly referred to throughout the API descriptions.
+
+    * __Application Provider__ - An application developer, onboarding his/her edge application on operator platform (MEC).    
+    * __Artifact__ - Descriptor, binary image, charts or any other package assosiated with the application.
+    * __OP__ - Operator Platform.
+    * __Region__ - Human readable name of the geografical zone of the MEC. Defined by the OP.
+    
+    # API Functionality
+    __Application Management__
+    * __onboardApplication__ - Submits an application details to an OP. Based on the details provided,  OP shall do bookkeeping, resource validation and other pre-deployment operations. 
+    * __removeApplication__ - Removes an application from an OP
+
+    __Application Artifacts Management__
+    * __onboardArtifact__  Submits an artifact to operator platform
+    * __removeArtifact__  Removes an artifact from operator platform
+    
+    __MEC Platform information__
+    *  __retrieveEdgeNodes__ List of the operator’s MECs and their status, ordering the results by location and filtering by status (active/inactive/unknown)
+       
+    # Further info and support
+  
+    (FAQs will be added in a later version of the documentation)
+
+  termsOfService: http://swagger.io/terms/
+  contact:
+    email: sp-edc@lists.camaraproject.org
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+  version: 1.0.0
+externalDocs:
+  description: Product documentation at Camara
+  url: https://github.com/camaraproject/
+
+servers:
+  - url: "{apiRoot}/{basePath}"
+    variables:
+      apiRoot:
+        default: http://localhost:9091
+        description: API root
+      basePath:
+        default: edge-cloud/v1
+        description: Base path for the Edge Cloud API
+
+security:
+  - oAuth2ClientCredentials: 
+    - nbi-mgmt
+
+tags:
+  - name: Applications
+    description: Manage of Applications
+  - name: Application Artifacts 
+    description: Manage of Artifacts
+  - name: Mobile Edge Cloud Platform
+    description: Mobile Edge Nodes Available
+
+paths:
+  /app:
+    post: 
+      tags:
+        - Applications
+      summary: Deploy an application 
+      description: Ask the operator to provision an application to one or several Edge Application Servers taking into account resources (e.g. vCPU, Memory, network interfaces, storage, GPU) 
```

Comment:
Always for clarity, we might want to also enforce the distinction between Application and Application instance. I see that the document already does it, but partially. But app for discussion.

Reply:
As discussed in the previous CAMARA - Edge Cloud meeting (12th - Dec) we agree to combine Artifact and Application into a single type, included in this new version. 

We clarified the difference between Application and Application Instance, defining two methods. Application is HOW to deploy meanwhile Application Instance is WHERE to deploy it. v0.2.0